Fabric.Authorization.API/web.config

```diff
@@ -1,38 +1,38 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
-    <system.webServer>
-        <security>
-            <authentication>
-                <anonymousAuthentication enabled="true" />
-            </authentication>
-            <requestFiltering allowDoubleEscaping="true"></requestFiltering>
-        </security>
-        <modules>
-            <remove name="WebDAVModule" />
-        </modules>
-        <rewrite>
-            <rules>
-                <rule name="allow backslash" stopProcessing="true">
-                    <match url="(.*)" />
-                    <action type="Rewrite" url="{C:1}{C:2}{C:3}" logRewrittenUrl="true" appendQueryString="false" />
-                    <conditions>
-                        <add input="{UNENCODED_URL}" pattern="(.*)(%5C)(.*)" />
-                    </conditions>
-                </rule>
-            </rules>
-        </rewrite>
-        <handlers>
-            <remove name="WebDAV" />
-            <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
-        </handlers>
-        <aspNetCore processPath="dotnet" arguments=".\Fabric.Authorization.API.dll" stdoutLogEnabled="true" stdoutLogFile=".\logs\stdout" >
-            <environmentVariables>
-                <clear />
-                <environmentVariable name="HostingOptions__UseIis" value="true" />
-                <environmentVariable name="MigrateDuplicateGroups" value="true" />
-                <environmentVariable name="MigrateGroupSource" value="true" />
-                <environmentVariable name="MigrateGroupIdentityProvider" value="true" />
-            </environmentVariables>
-        </aspNetCore>
-    </system.webServer>
+  <system.webServer>
+    <security>
+      <authentication>
+        <anonymousAuthentication enabled="true" />
+      </authentication>
+      <requestFiltering allowDoubleEscaping="true"></requestFiltering>
+    </security>
+    <modules>
+      <remove name="WebDAVModule" />
+    </modules>
+    <rewrite>
+      <rules>
+        <rule name="allow backslash" stopProcessing="true">
+          <match url="(.*)" />
+          <action type="Rewrite" url="{C:1}{C:2}{C:3}" logRewrittenUrl="true" appendQueryString="false" />
+          <conditions>
+            <add input="{UNENCODED_URL}" pattern="(.*)(%5C)(.*)" />
+          </conditions>
+        </rule>
+      </rules>
+    </rewrite>
+    <handlers>
+      <remove name="WebDAV" />
+      <add name="aspNetCore" path="*" verb="*" modules="AspNetCoreModule" resourceType="Unspecified" />
+    </handlers>
+    <aspNetCore processPath="bin\IISSupport\VSIISExeLauncher.exe" arguments="-argFile IISExeLauncherArgs.txt" stdoutLogEnabled="true" stdoutLogFile=".\logs\stdout">

```

Comment:
I think this changed because of Mike's dev environment changes. Josh noticed this in his PR and we reverted it. Do we want this change permanently?

Reply:
reverted